RELEASE.md

```diff
@@ -155,39 +151,45 @@ point but for now the process is manual.
 
 ## Setup
 
-1. You must have access to the `SFX_Windows.pfx` code signing certificate and password.
+1. You must be on the Splunk network and have access to the
+   `repo.splunk.com Artifactory Token` and `srv-signalfx-agent Chaperone Token`
+   service account secrets in [Thycotic](https://secret.splunk.com) to sign the agent
+   executable.
+
+1. You must have a Github access token to publish the agent bundle to Github Releases.
 
 1. You must have your AWS CLI set up on your local workstation and have access to our
    S3 bucket.
 
-1. Build, provision, and start the Windows Server 2008 vagrant box. See the "Windows"
-   section in [development.md](docs/development.md) for details.
-
-1. Copy `SFX_Windows.pfx` to some folder in the virtual machine.
+1. You must have access to a Windows machine that is provisioned with the required
+   build tools.  Alternatively, you can build, provision, and start the Windows
+   Server 2016 vagrant box. See the "Windows" section in
+   [development.md](docs/development.md) for details.
 
 ## Release Process
 
 1. Open a Powershell terminal in the Windows virtual machine and execute:
 
    ```
    $ cd c:\users\vagrant\signalfx-agent
-   $ scripts/windows/make.ps1 bundle -AGENT_VERSION "<X.Y.Z>" -PFX_PATH "<PFX_PATH>" -PFX_PASSWORD "<PFX_PASSWORD>"
```

Comment:
Why does this go away?

Reply:
> Why does the PFX path go away?

We will be signing the windows agent with the splunk key, so the PFX will no longer be required.